### DISCUSSION TITLE
My feed is broken / stuck in time

### DISCUSSION THREAD

**Discussion by SaschaWillems** [community/community#67514] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The new feed is a huge step back and is utterly broken. No matter what settings I enable, it is stuck in the past. It tells me that the latest star to one of my repos is 3 weeks old, which is 100% not the case. There have been stars recently. And even though the new feed takes ages to load, all it does is display broken/outdated data:

![image](https://github.com/community/community/assets/10283127/7f99f1b2-bc5f-4453-9a22-c288522d2bcb)

Any help on how to fix this this from the github staff that worked on this would be appreciated.

**Comment by Col-E** [comment]
I also have been experiencing this for the past ~2 weeks, however it unstuck itself to show me something I am not following and don't want in my feed anyways.

![image](https://github.com/community/community/assets/21371686/4aaebdf6-bc89-4b0d-b003-c1d21b7927e4)

There's dozens of other threads with _"only show me what I follow"_ so I don't really need to blab about that here. Just noting the somewhat unstuckness.

**Reply by Col-E** [reply]
Back to being stuck. This unwanted entry was just a one-off break from being totally silent on updates :/

**Comment by muddlebee** [comment]
the github experience is degrading day by day

**Comment by baggiponte** [comment]
Me too, I tried to filter only for stars and repositories and the only posts that appear are some people that a couple of weeks ago starred a repo of mine. We just want the old feed back.

**Comment by ElectNewt** [comment]
same here, no update since the new useless feed was forced for everyone.

**Comment by woheller69** [comment]
I don't get updates either. Since last update of the feed I have received about 40 stars which are not shown...
